WebStoreTerehin/Controllers/EmployeesController.cs

```diff
@@ -78,12 +163,12 @@ public IActionResult AddEmp(string surname, string name, string patronymic, stri
                 return NotFound();
 
             int id;
-            if (_Employees.Count() == 0)
+            if (TestData.Employees.Count() == 0)
```

Comment:
```suggestion
            if (TestData.Employees.Count == 0)
```
Незачем лишний раз призывать демона Linq